src/DebuggerForm.cpp

```diff
@@ -1136,20 +1136,21 @@ void DebuggerForm::executeStepBack()
 	setRunMode();
 }
 
-void DebuggerForm::toggleBreakpoint(int addr)
+void DebuggerForm::toggleBreakpoint(std::optional<uint16_t> addr)
```

Comment:
As far as I can see you call this method:
* Either with no parameters (which defaults to an empty optional).
* Or with an `int` parameter (which constructs a non-empty optional).

(I'm not 100% sure because it's via signal/slots).

It's more idiomatic to instead create two overloaded versions of this method. And then have the version without parameters delegate to the version with an `int` parameter.